plenigo_js/pl_settings.js

```diff
@@ -0,0 +1,105 @@
+var plenigoSettings = plenigoSettings || {
```

Comment:
Non linked yet, it is part of the new editor. I needed to bring some stuff from the other branch and this got caught.